source-manager/src/main/java/com/jointrivial/sourcemanager/nordigen/model/view/KeyOrganisationViewModel.java

```diff
@@ -0,0 +1,28 @@
+package com.jointrivial.sourcemanager.nordigen.model.view;
+
+public class KeyOrganisationViewModel {
+
+    private String institutionId;
+    private String accountId;
```

Comment:
change field name to organizationKey

Reply:
Same as the upper

Reply:
Again - consistency. Not a big deal anyway..